api/src/routing.js

```diff
@@ -12,15 +12,28 @@ const prometheusMiddleware = require('prometheus-api-metrics');
 const rateLimiterMiddleware = require('./middleware/rate-limiter');
 const logger = require('./logger');
 const isClientHuman = require('./is-client-human');
-const target = 'http://' + environment.host + ':' + environment.port;
-const proxy = require('http-proxy').createProxyServer({ target: target });
+
+const isString = value => typeof value === 'string' || value instanceof String;
+const isTrue = value => isString(value) ? value.toLowerCase() === 'true' : value === true;
+
+const { 
+  PROXY_CHANGE_ORIGIN = false,
+  PROXY_CHANGE_ORIGIN_AUTH = false,
+  PROXY_CHANGE_ORIGIN_CHANGES = false
+} = process.env;
```

Comment:
To make it easier to reuse these, or at least see what options are available, I think these should go in the environment.js

Reply:
Also can you add more info about what these three are for? Surely if you want to `changeOrigin` for one, you want to do it for all 3?

Reply:
Will look into it.

Reply:
> Surely if you want to changeOrigin for one, you want to do it for all 3?

I didn't want to foreclose on optionality here, so I made each client individually configurable. Would you prefer collapsing to one variable or having an override that sets them all while retaining the granularity?

Reply:
Yeah let's not add complexity that we don't know there's a requirement for it. Stick with 1 if that works for your use case, and it'll be easy to add the other two if we have a good reason to.